apps/admin/src/features/group-management/model/useGroupMembersField.ts

```
@@ -0,0 +1,101 @@
+
+      return nextLeader;
+    });
+  }, []);
+
+  /** 여러 명 추가 (중복 제거) */
+  const addMembers = useCallback((newMembers: MemberBase[]) => {
+    setMembers((prev) => {
+      const map = new Map<Id, MemberBase>();
+      prev.forEach((m) => map.set(m.id, m));
+      newMembers.forEach((m) => map.set(m.id, m));
+      return Array.from(map.values());
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)